src/StaticWebAssetsSdk/Targets/Sdk.StaticWebAssets.CurrentVersion.targets

```diff
@@ -31,7 +31,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_StaticWebAssetsSdkTasksTFM Condition=" '$(MSBuildRuntimeType)' == 'Core'">net8.0</_StaticWebAssetsSdkTasksTFM>
     <_StaticWebAssetsSdkTasksTFM Condition=" '$(_StaticWebAssetsSdkTasksTFM)' == ''">net472</_StaticWebAssetsSdkTasksTFM>
     <StaticWebAssetsSdkBuildTasksAssembly>$(StaticWebAssetsSdkBuildTasksDirectoryRoot)$(_StaticWebAssetsSdkTasksTFM)\Microsoft.NET.Sdk.StaticWebAssets.Tasks.dll</StaticWebAssetsSdkBuildTasksAssembly>
-    <_StaticWebAssetsSdkToolAssembly>$(StaticWebAssetsSdkDirectoryRoot)..\Microsoft.NET.Sdk.Razor\tools\rzc.dll</_StaticWebAssetsSdkToolAssembly>
+    <_StaticWebAssetsSdkToolAssembly>$(StaticWebAssetsSdkDirectoryRoot)tools\net8.0\Microsoft.NET.Sdk.StaticWebAssets.Tool.dll</_StaticWebAssetsSdkToolAssembly>
```

Comment:
Is it correct to always have net8.0 here?

Reply:
We do something similar in [Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets](https://github.com/dotnet/sdk/blob/0961189a48b0763d7cccba637539c876ed29f43f/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets#L33), where we've just been updating the TFM manually. Not sure why we haven't been parameterizing it. @javiercn what do you think?

Reply:
I think so, since this gets put in the SDK, we don't want to evaluate the condition based on the user target framework (that would be wrong) and we don't have a mechanism to pick it up automagically from the TFM the project is targeting, so I think having this hardcoded is ok.

It will break immediately when we update the repo branding.